opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -629,8 +667,9 @@ mod tests {
     }
 
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    #[ignore = "Spatial aggregation is not yet implemented."]
```

Comment:
this got accidentally fixed in some other PR, but this PR breaks it. spatial aggregation is complex and needs a separate discussion altogether!

Reply:
If there is no issue to track this, probably good to have one.